.github/workflows/doc-tests.yaml

```
@@ -53,6 +53,27 @@ jobs:
           repository: 'gravitational/docs-website'
           path: 'docs'
 
+      - name: Generate GitHub Token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.REVIEWERS_APP_ID }}
+          private-key: ${{ secrets.REVIEWERS_PRIVATE_KEY }}
+
+      - name: Check out shared-workflows
+        uses: actions/checkout@v4
+        with:
+          repository: gravitational/shared-workflows
+          path: .github/shared-workflows
+
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 'stable'
+
+      - name: Ensure docs changes include redirects
+        run: cd .github/shared-workflows/bot && go run main.go -workflow=docpaths -token="${{ steps.generate_token.outputs.token }}" -teleport-path="${{ github.workspace }}/teleport" -reviewers="${{ secrets.reviewers }}"
```

```suggestion
        env:
          TOKEN: ${{ steps.generate_token.outputs.token }}
          REVIEWERS: ${{ secrets.reviewers }}
        run: cd .github/shared-workflows/bot && go run main.go -workflow=docpaths -token="${TOKEN}" -teleport-path="${GITHUB_WORKSPACE}/teleport" -reviewers="${REVIEWERS}"
```

This should probably use env vars instead of interpolation with GHA contexts. If the context vars contain a double quote or other shell value, then the resulting command will not be what you expect.

---

Thanks!